node/service_setup.go

```diff
@@ -110,6 +122,8 @@ func (node *Node) ServiceManagerSetup() {
 		node.setupForNewNode()
 	case nodeconfig.ClientNode:
 		node.setupForClientNode()
+	case nodeconfig.WalletNode:
```

Comment:
why WalletNode - thought after offline chat, we would go to unify ClientNode logic